pkg/lib/unlock.go

```diff
@@ -71,15 +53,12 @@ func UnlockAllWithLogger(tpm bool, log types.KairosLogger) error {
 							logger.Warn().Msgf("Unlocking failed, command output: '%s'\n", out)
 						}
 					} else {
```

Comment:
```
if tpm {
} else {
this_here
}
```

So if `tpm` is true we decrypt with `systemd-cryptsetup`. I wonder which user option this implements... Anyway, irrelevant.